feat(curriculum) : Created a count up workshop for Javascript certification (#64828)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-create-count-up/69446f18fdcb11aa516f7c5c.md

---
id: 69446f18fdcb11aa516f7c5c
title: Step 1
challengeType: 1
dashedName: step-1
demoType: onClick
---

# --description--

In this workshop, you will be creating a 'countUp' function.
Start by creating an empty function named 'countUp' that takes a single parameter 'number'.

> This function will later be used to build an array that count up from 1 to 'number'.

Reviewer: ```suggestion
In this workshop, you will be creating a `countUp` function.

Start by creating an empty function named `countUp` that takes a single parameter `number`.

This function will later be used to build an array that count up from 1 to `number`.
```

# --hints--

"Your function should be named 'countUp' and accept a single parameter"

```js

> assert(typeof countup === 'function');

Reviewer: ```suggestion
assert.isFunction(countup);
```

documentation on the assert methods https://www.chaijs.com/api/assert/

---
id: 69446f18fdcb11aa516f7c5c
title: Step 1
challengeType: 1
dashedName: step-1
---

# --description--

> In this workshop, you will be creating a `countUp` function.

Reviewer: ```suggestion
In this workshop, you will be creating a `countup` function.
```

please update the function name in the workshop
